andela/mmdp_site
# MMDP-341 Configure linting and refractor codebase to suit lint config

**What does this PR do?**
- Lints codebase following airbnb style guide and prettier config

**Description of Task to be completed?**
- [x] Add prettier plugin
- [x] Lint codebase
- [x] Add pretest command to run before test scripts

**How should this be manually tested?**

-  RUN`yarn lint` to run lint on codebase
-  RUN`yarn lint:fix` to run lint and fix autofixable issues on codebase
-  RUN `test:server` and observe that lint command runs before test command 
-  RUN `test:client` and observe that lint command runs before test command 

**Any background context you want to provide?**
- none

**What are the relevant Jira issues?**

[MMPD-341](https://viisaus.atlassian.net/browse/MMPD-341)

**Screenshots (if appropriate)**
- none

**Questions:**
- none
